ansible/roles/eos/templates/t1-lag-spine.j2

```diff
@@ -97,6 +97,9 @@ route-map PREPENDAS permit {{ 2 * (podset * props.tor_number + tor + 1) + 1 }}
 router bgp {{ host['bgp']['asn'] }}
  router-id {{ host['interfaces']['Loopback0']['ipv4'] |  ipaddr('address') }}
  !
+ graceful-restart restart-time {{ bgp_gr_timer }}
+ graceful-restart
+ !
```

Comment:
Is the possible to insert the changes before the test and remove them after the test?